tests/filecheck/dialects/llvm/generic-syntax.mlir

```diff
@@ -0,0 +1,61 @@
+// RUN: xdsl-opt %s --print-op-generic | filecheck %s
+
+"builtin.module"() ({
```

Comment:
The module can also be left implicit

Reply:
For the "This is the command XDSL_GENERIC_ROUNDTRIP right?" comment:

Yes it is, but if I read `config.substitutions.append(("XDSL_GENERIC_ROUNDTRIP", "xdsl-opt %s --print-op-generic --split-input-file | filecheck %s --check-prefix=CHECK-GENERIC"))` correctly,
that command doesn't do the roundtrip Mathieu suggested

Reply:
Fair enough, go with Mathieu's suggestion instead then